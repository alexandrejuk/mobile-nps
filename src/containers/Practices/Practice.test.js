import React from 'react'
import { render } from '@testing-library/react'

import PracticesContainer from '.'

const practices = [
  {
    id: 1,
    image: 'BannerSvg3',
    title: "Resumo dos Procedimentos para Auditoria TCP",
    readTime: '5-6 minutos de leitura',
  }
]

describe('Practices container', () => {
  it('should render correctly', () => {
    const callback = jest.fn()
    const { container, getByText } = render(
      <PracticesContainer
        goBack={callback}
        goToPractice={callback}
        practices={practices}
      />
    )

    getByText('Read more')
    expect(container).toMatchSnapshot()
  })
})
