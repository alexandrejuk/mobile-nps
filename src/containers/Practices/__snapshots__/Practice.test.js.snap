// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Practices container should render correctly 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="header"
    >
      <div
        role="button"
      >
        <img
          alt="go back"
          src="arrow-left-white.svg"
        />
      </div>
      <img
        alt="ilustration"
        src="banner-2.svg"
      />
    </div>
    <div
      class="content"
    >
      <h1
        class="primary medium left title bold"
      >
        Read more
      </h1>
      <div
        class="pratices"
      >
        <div
          class="card"
          role="button"
          tabindex="0"
        >
          <div
            class="imageContainer"
          >
            <img
              alt="item card"
              src="banner-3.svg"
            />
          </div>
          <div
            class="content"
          >
            <h1
              class="primary medium left title bold"
            >
              Resumo dos Procedimentos para Auditoria TCP
            </h1>
            <h1
              class="secondary small right title mediumBold"
            >
              5-6 minutos de leitura
            </h1>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
